lib/internal/http2/util.js

```diff
@@ -415,16 +417,18 @@ function mapToHeaders(map,
       if (isArray) {
         for (var k = 0; k < value.length; k++) {
           val = String(value[k]);
-          ret.push([key, val]);
+          ret += `${key}\0${val}\0`;
         }
+        count += value.length;
       } else {
         val = String(value);
-        ret.push([key, val]);
```

Comment:
@jasnell That depends – it’s pretty easy to add, and like others I don’t like having unused, untested code lying around. Is it okay if I give the API issue some thoughts and see if I can take care of this myself?